xsdata/writer.py

```diff
@@ -28,20 +31,46 @@ def get_format(self, name) -> AbstractGenerator:
             return self.generators[name]
         raise CodeWriterError(f"Unknown code generator `{name}`")
 
-    def write(self, schema: Schema, classes: List[Class], package: str, format: str):
+    def write(self, classes: List[Class], format: str):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'format'](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5141157)